_data/assetPaths.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
We shouldn't be committing this file. Can you please remove this file from the commit? (I or any other engineer can help with this, if needed)

Reply:
This is a stupid thing. It really is.

This is an issue specific to 11ty and `assetPaths.json`.

Basically, we should always be ignoring this file, _but_ if we git ignore this file, then the .gitignore change prevents changes to scss files from triggering a build -- which just means that styles are not linked properly and cause weird issues in the outputted html.

So if we can't ignore it or if we can't commit it, what do we do? We'll have to exclude it from the git repository. 

You can exclude by doing:

1. Open up .git/info/exclude
2. Add `assetPaths.json` to that file

Or, the simpler solution is not to include `assetPaths.json` in the PR. 

@caleywoods we may need a better solution than this for newer folks. Any ideas?

Reply:
I suggest adding `_data/assetPaths.json`, but I don't know if it's strictly necessary.

As an FYI, this was working for a while for me, but in my last PR, I found that the file got added even though it's in my .git/info/exclude. No idea how that's possible, what happened, or how to fix it.

Reply:
Oof. Yea. `_data/assetPaths.json` is more precise -- thanks for the catch.

In either case, this "solution" is a bit too fiddly for my taste. We're not the only one impacted by this edge case, either. See: https://github.com/cisagov/manage.get.gov/issues/705

Reply:
I removed the `assetPaths` from the PR.